eslint.config.ts

```diff
@@ -25,10 +27,10 @@ export default tseslint.config([
   ...compat.extends(
     'not-an-aardvark/node',
     'plugin:@eslint-community/eslint-comments/recommended',
-    'plugin:prettier/recommended',
     'plugin:unicorn/recommended',
   ),
   pluginN.configs['flat/recommended'],
+  prettier,
```

Comment:
I think we technically want to include this last to ensure it overrides any incompatible config?

> eslint.config.js (flat config): Import eslint-config-prettier, and put it in the configuration array – after other configs that you want to override.

Reply:
I will generally put the prettier config after all of the reusable configs, but before any configs that I've created with overrides / additional rules.  The idea being that anything I add myself is an intentional choice and should take precedence, which I think aligns with the spirit of that documentation.  With that said, it's currently before the TS Configs, so I should at least move it to be after those (I authored this PR before adding the TS configs).  Would you prefer I move it all the way to the end or just after the ts configs?

Reply:
Your reasoning makes sense. I'll leave it up to you.